src/components/Title/style.css

```diff
@@ -31,8 +32,8 @@
   z-index: 1;
   .title {
     color: white;
-    font-size: 16px;
+    font-size: 16*@unit;
     font-weight: 400;
-    line-height: 24px;
+    line-height: 24*@unit;
   }
```

Comment:
I think you need to add `.titleSpan { font-size: 16*@unit; }` which will apply the right units to the titles nested within the `.fullScreen` class

Reply:
Yes, good catch :) Fixed this up now